arangod/Cluster/FollowerInfo.cpp

```diff
@@ -305,11 +305,13 @@ bool FollowerInfo::updateFailoverCandidates() {
 // All followers can return as soon as the lock is released
 #ifdef ARANGODB_ENABLE_MAINTAINER_MODE
     TRI_ASSERT(_failoverCandidates->size() == _followers->size());
+#if 0
```

Comment:
I guess the changes in this file are temporary workarounds and should be removed from this PR.

Reply:
Yes, there is a PR out there which fixes this and once merged, this will work again.